commonjs-extension-resolution-loader/test/fixtures/.gitignore

```diff
@@ -0,0 +1 @@
+!node_modules/
```

Comment:
Why do you need that, since you have a lockfile?

Reply:
Those fixtures are fake modules, not packages on npm. Look under `test/fixtures`, it's lots of variations of types of `package.json`.

Reply:
Ah that makes sense, lgtm then 👍